CHANGELOG.md

```diff
@@ -1,5 +1,188 @@
 # Changelog
 
+## 🔎 1.1.0
+
+Wrangler 1.1.0 includes a number of improvements to documentation and project stability, including: 
+
+- ### Features
+
+  - **Add message module for terminal output - [ashleymical], [issue/219] [pull/263]**
+
+    We've made improvements to Wrangler's terminal output functionality, with support for various log levels and implementations in Wrangler's API for easily using the log levels in future development.
+
+    [ashleymical]: https://github.com/ashleymichal
+    [issue/219]: https://github.com/cloudflare/wrangler/issues/219
+    [pull/263]: https://github.com/cloudflare/wrangler/pull/263
+
+  - **Adds more descriptive subdomain errors - [EverlastingBugstopper], [issue/207] [pull/259]**
+
+    Warns users when attempting to add a subdomain w/o account_id in wrangler.toml
```

Comment:
i'd probably share the scenario and what . the . error message is now... "lots of folks will first use wrangler and forget to add this.. and then this happened.. and it was confising!! so we decided to do this.. it looks like this now"

Reply:
Previously, when attempting to register the subdomain that was already associated with your account, wrangler would error something like “That subdomain is already registered” and now it’s something like “You’ve already registered that subdomain”